diff --git a/base/BUILD.gn b/base/BUILD.gn
index ba907d51ea11537e83a6c7c813c13ec51da808c0..224bd00f24afea405ccf29a8dc134db192e84e21 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4402,6 +4402,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
