diff --git a/base/BUILD.gn b/base/BUILD.gn
index 83cdc61ae5ac4b98fc1d68ad99672606c75ddd09..5ed1d99313a7d70d7ba8f3d2dfe8f3d77f811c2a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4394,6 +4394,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
