diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index d00ba9be7ab0d2002bb4983ccd8e2dbf7b508807..a38352125f9e3c3f83881b22f1775f9ac3686b13 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -316,6 +316,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.disable_checks:
       cmd += ['--map-diagnostics:CheckDiscardDiagnostic', 'error', 'none']
