diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index a56e003a44a181fd4ecb6c4292e984106b980fa0..52224d09e02a05ff922dc1e65215464add98d0f4 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -404,6 +404,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
