diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 82727744a2b8b37a65b999e7ec23997eef8b5f98..eb1fc4b832b6d6c9d7e5016de660520937da9bcc 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2553,6 +2553,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
