diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index f6e8fb28ceb42d758bf90a4c54f1d1b1005ba78f..eb0c9dbc5026c00b617d2a55aeab5443db0afc59 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -224,6 +224,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
