diff --git a/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc b/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
index b9e437ec5121614116eee16d0409cd96777a14b3..8e5c600f9fa4c3abb30279912a56ba967d58e8b7 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
@@ -353,7 +353,7 @@ void OmniboxPopupViewViews::UpdatePopupAppearance() {
       if (children().size() == i) {
         AddChildView(std::make_unique<OmniboxRowView>(
             i, edit_model_,
-            std::make_unique<OmniboxResultView>(this, edit_model_, i),
+            std::make_unique<BraveOmniboxResultView>(this, edit_model_, i),
             pref_service));
       }
 
