diff --git a/chrome/browser/resources/settings/settings.ts b/chrome/browser/resources/settings/settings.ts
index 25f9ab700186c252e98fb97e56fc1d949f41bc69..03f00c0d796f9bd9dfafa730b08053cfbd3a5dcd 100644
--- a/chrome/browser/resources/settings/settings.ts
+++ b/chrome/browser/resources/settings/settings.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './settings_ui/settings_ui.js';
 
 export {ControlledRadioButtonElement} from '/shared/settings/controls/controlled_radio_button.js';
