diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index c98f4182718e57bc9f6a9d2f1a3297ab0ffe9212..69bbe08311a04f4a5583203709f8a473b4acb6fd 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -345,6 +345,7 @@ void BookmarkBubbleView::ShowBubble(
       dialog_model_builder.Build(), anchor_view,
       views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
