diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index 10ca98bf89fdd00b4ef3c4b128a41fe228705057..8ba4abfb3d53a561a8ac7d7c87240879eb2dbf3d 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -133,7 +133,7 @@ void ThemeSource::StartDataRequest(
   int resource_id = -1;
   if (parsed_path == "current-channel-logo") {
     switch (chrome::GetChannel()) {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
       case version_info::Channel::CANARY:
         resource_id = IDR_PRODUCT_LOGO_32_CANARY;
         break;
@@ -155,7 +155,10 @@ void ThemeSource::StartDataRequest(
         [[fallthrough]];
 #endif
       case version_info::Channel::UNKNOWN:
+        resource_id = IDR_PRODUCT_LOGO_32_DEVELOPMENT;
+#if 0
         resource_id = IDR_PRODUCT_LOGO_32;
+#endif
         break;
     }
   } else {
