diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 0fe347872e1e3f4731b6b6d8cd7150e724c6bdaa..1e256af9af0bf5aa06b8eb0849a9b535be970fd7 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -124,6 +124,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
