diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index f62c0f68ae2f51b3a669336d2d22c0050abec8f5..f53ed97ecc6456b886a5762fb53c6145afebb0ae 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -384,6 +384,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_placement";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
