diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 6de506103393cb5ad2e05d60f5947d9b4592a9c7..5de539551670d52625e093020d93b07ca5246f82 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -248,6 +248,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
