diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index c3903f86ef648132a2b8856cf8181c5fa7c9d799..5a3e449c2d26de766ef3169ff048932db066080c 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -376,6 +376,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
