diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index a2fda2aefc88cca8652720f2fe31874d808a427c..7e3c5273e6d8158afd480c5008139a2c7955065e 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -160,6 +160,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/engine/sources.gni") sources += brave_components_sync_engine_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
