diff --git a/components/content_settings/core/browser/cookie_settings.cc b/components/content_settings/core/browser/cookie_settings.cc
index 383fdd336c77e896697fc7d65d927cca7a393bd9..579b9cfd36d0b710cb9fdc4d2085fe47ee75d52d 100644
--- a/components/content_settings/core/browser/cookie_settings.cc
+++ b/components/content_settings/core/browser/cookie_settings.cc
@@ -234,6 +234,7 @@ ContentSetting CookieSettings::GetCookieSettingInternal(
   DCHECK(value.is_int());
   ContentSetting setting = ValueToContentSetting(value);
   bool block = block_third && is_third_party_request;
+  BRAVE_COOKIE_SETTINGS_GET_COOKIES_SETTINGS_INTERNAL
 
   if (!block) {
     FireStorageAccessHistogram(
