diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index c493bb95b63d2d0810bfe50747e75dbbb0e4319c..ab0a77b1c965ab8778d1586063ea6c9bcb8a805c 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4487,6 +4487,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
