diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 9e9a06ef58e9aeff8193df7b198ee69939c2dd8f..4f838e6dc4f18fef9ebf010ad039128bdc68c464 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
