diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 2326ec6a672051507dfdad076df9d4805e24a34d..aa2513b597582d623ae42154db25c11f5d9b36b0 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1203,6 +1203,7 @@ static_idl_files_in_modules = get_path_info(
           "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
         ],
         "abspath")
+static_idl_files_in_modules += get_path_info(brave_static_idl_files_in_modules, "abspath")
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
