diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 26dcc1cde2d2f33a90dabec4cb9e4217f14b9736..e7b187acc9d5992b213a2213b642ce9b52979947 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -2020,6 +2020,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -2160,6 +2161,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
       DCHECK(!bounds.intersect(SkIRect::MakeXYWH(sx, sy, sw, sh)));
     }
   }
+  BRAVE_GET_IMAGE_DATA
 
   return image_data;
 }
