diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index ab8758afdc563362415ccce5fb25d496c3e28be3..e8bbfc5b93e534a5fad895a84cbb83e9e3a65a52 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2935,6 +2935,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
