diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index d8b29efcf25fe7a3633af45c0c94cbc940975f0f..7038d41dd5a4f3578a5465febdf79b3c7149f366 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2921,6 +2921,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
